webpack.config.js

```diff
@@ -66,4 +69,20 @@ module.exports = {
 			}
 		]
 	},
+
+	plugins: [
+		new CopyPlugin( {
+			patterns: [
+				{
+					from: resolvePath( __dirname, 'node_modules', 'ckeditor4' ),
+					to: resolvePath( distPath, 'ckeditor4' )
+				},
+
+				{
+					from: resolvePath( srcPath, 'index.html' ),
+					to: distPath
+				}
+			]
```

Comment:
I wonder if we need to copy entire `ckeditor4` directory? The bundled `dist/app.js` already contains e.g. `ckeditor.js` file AFAIU? So we probably need only files which are loaded by CKEditor 4 during runtime (config, lang files, plugins files).

It's easier to copy entire `ckeditor4` folder but I'm not sure if necessary. Also it might be confusing since some files are already bundled, why we would need to copy team once again?